samples/subsys/usb/mass/prj_nrf52840_pca10056.conf

```diff
@@ -0,0 +1,12 @@
+CONFIG_STDOUT_CONSOLE=y
```

Comment:
Is this file necessary? It would fit a lot of boards and it is not that it is tuned for pca10056. The same configuration to build is also given in `prj.conf` and `overlay-ram-disk.conf` or do I miss something?

Reply:
For now it is `proj.conf` and `overlay-ram-disk.conf` combined. After #14459 is fixed, I want to add `overlay-flash-disk.conf` too. Is there a way to define a .conf file specific to multiple boards?

Reply:
Do we really need to duplicate those 2 configuration files?

Reply:
Can you do this in cmake files? @carlescufi